### PR TITLE
Made it possible to switch UI to english in development

### PR DESCRIPTION
**Pull request by damisul**
If BY_LANGUAGE env variable is provided it will try to use given locale as default.
For now only en or he locale is accepted (and only in dev environment)

I've added dotenv gem to simply manage env variables in development and test. I plan to use it also in https://github.com/abartov/bybeconv/issues/166

You can create `.env.development.local` file and override BY_LANGUAGE value there instead of overwriting `.env.development`

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -0,0 +1 @@
+BY_LANGUAGE=he
```
Comment (damisul) [inline .env.development:1]: provided as an example for dotenv config. To override it locally create a new file .env.development.local (should not be committed to git)

File: config/environments/production.rb [hunk 2]
```diff
@@ -68,16 +68,11 @@
   # Enable threaded mode
   # config.threadsafe!
 
-  # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
-  # the I18n.default_locale when a translation can not be found)
-  config.i18n.fallbacks = true
```
Comment (damisul) [inline config/environments/production.rb:73]: already specified in application.rb

File: config/environments/production.rb [hunk 3]
```diff
@@ -68,16 +68,11 @@
...
-
   # Send deprecation notices to registered listeners
   config.active_support.deprecation = :notify
 
   config.action_mailer.delivery_method = :sendmail # let's start sending mail :)
   config.eager_load = true
-  config.i18n.available_locales = :he
```
Comment (damisul) [inline config/environments/production.rb:80]: already specified in application.rb

File: config/environments/production.rb [hunk 4]
```diff
@@ -102,10 +97,6 @@
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
 
-  # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
-  # the I18n.default_locale when a translation cannot be found).
-  config.i18n.fallbacks = true
```
Comment (damisul) [inline config/environments/production.rb:107]: duplicate line

File: config/environments/development.rb [hunk 5]
```diff
@@ -54,7 +55,6 @@
   #config.assets.debug = true
   config.eager_load = false
   # config.public_file_server.enabled = true # Rails 5.x?
-  config.i18n.available_locales = :he
```
Comment (damisul) [inline config/environments/development.rb:57]: defined in application.rb

File: config/locales/en.yml [hunk 6]
```diff
@@ -1,4 +1,5 @@
 en:
+  edit_metadata: Edit metadata
```
Comment (damisul) [inline config/locales/en.yml:2]: added for tests

**Review by abartov — APPROVED** [review]

**Pull request merged**